index.js

```diff
@@ -182,9 +182,21 @@ async function scrapeData() {
   return { locations };
 };
 
+function calculateActive(locations) {
+  return locations.map(l => {
```

Comment:
Can we move this inner arrow function to `lib/transform.js` called `getActiveFromLocation`?

Reply:
fixed